src/material/core/theming/_theming-deprecated.scss

```diff
@@ -0,0 +1,27 @@
+@use './theming';
+@use './palette';
+
+// @deprecated Use `get-color-from-palette`.
+@function color($palette, $hue: default, $opacity: null) {
+  @return get-color-from-palette($palette, $hue, $opacity);
```

Comment:
This is missing the module name (caught by the presubmit I ran, surprised it didn't fail here)

Reply:
Fixed.